test/claim-iat.test.js

```diff
@@ -178,6 +157,24 @@ describe('issue at', function() {
         });
       });
     });
+
+    it('should throw exception when given -Infinity', function () {
+      expect(() => signWithIssueAtSync({iat: -Infinity})).to.throw(
+        '"iat" should be a number of seconds'
+      );
+    });
+
+    it('should throw exception when given Infinity', function () {
+      expect(() => signWithIssueAtSync({iat: Infinity})).to.throw(
+        '"iat" should be a number of seconds'
+      );
+    });
+
+    it('should throw exception when given NaN', function () {
+      expect(() => signWithIssueAtSync({iat: NaN})).to.throw(
+        '"iat" should be a number of seconds'
+      );
+    });
```

Comment:
These test can be moved to the array above in the `'jwt.sign "iat" claim validation'` block.